.github/company.js

```diff
@@ -0,0 +1,128 @@
+let querySymbol = new URLSearchParams(window.location.search);
+let symbol = querySymbol.toString()
+symbol = symbol.slice(7)
+
+let url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${symbol}`
+function companydata() {
+    displayLoader()
+fetch(url).then(response => {
```

Comment:
you need to choose between async await and .then syntax and keep with it everywhere in your code